apiserver/gui.go

```diff
@@ -354,7 +364,7 @@ func (h *guiHandler) serveConfig(w http.ResponseWriter, req *http.Request) {
 	w.Header().Set("Content-Type", jsMimeType)
 	tmpl := filepath.Join(h.rootDir, "templates", "config.js.go")
 	if err := renderGUITemplate(w, tmpl, map[string]interface{}{
-		"base":             h.baseURLPath,
+		"base":             "/gui/",
```

Comment:
can you please make this a const, and use that in the guiEndpoints calls in apiserver.go?